pytradfri/command.py

```diff
@@ -119,15 +119,15 @@ def _merge(
                 b_dict[key] = value
         return b_dict
 
-    def combine_data(self, command2: Command | None) -> None:
+    def combine_data(self, command2: Command[T] | None) -> None:
```

Comment:
I'm not sure that the parameter is limited to a command with the same return value as this command. Maybe put `Any` instead of `T` for now.